.circleci/config.yml

```diff
@@ -57,13 +57,18 @@ jobs:
       - store_artifacts:
           path: build/libsolc/soljson.js
           destination: soljson.js
+      - store_artifacts:
+          path: build/libsolc/soljson.wasm
+          destination: soljson.wasm
```

Comment:
I'd call this `libsolc.wasm` tbh.

Reply:
Fine with me - ``soljson.wasm`` is what emscripten outputs without telling it otherwise, so I just kept it.